apis/python/src/tiledbsoma/__init__.py

```diff
@@ -1,3 +1,4 @@
+from . import experiment_query
```

Comment:
This doesn’t seem like it should be necessary, since `from tiledbsoma import experiment_query` or `import tiledbsoma.experiment_query` will work without it.